### PR TITLE
Add game_started flag and search for 'TeamAbbrev' substring

### PR DESCRIPTION
**Pull request by utahman**

### CODE REVIEW

**Comment by utahman** [comment]
Ready

File: pydfs_lineup_optimizer/sites/draftkings/classic/importer.py [hunk 1]
```diff
@@ -48,14 +55,17 @@ def _row_to_player(self, row):
...
-            first_line = csv_file.readline()
-            csv_file.seek(0)
-            if 'TeamAbbrev' in first_line:
-                csv_data = csv.DictReader(csv_file, skipinitialspace=True)
-            else:
-                csv_data = csv.DictReader(islice(csv_file, 7, None),
-                                          skipinitialspace=True)
-            players = [self._row_to_player(row) for row in csv_data]
+            start_line = 0 # Find line with 'TeamAbbrev', that's where players data starts
+            while True:
```
Comment (DimaKudosh) [inline pydfs_lineup_optimizer/sites/draftkings/classic/importer.py]: If user pass incorrect file without TeamAbbrev program will be in infinity loop. Why do you change this method? From csv samples TeamAbbrev placed in first or eighth row.

File: pydfs_lineup_optimizer/sites/draftkings/classic/importer.py [hunk 2]
```diff
@@ -77,6 +87,8 @@ def import_lineups(self, players):
                 for index, position in zip(range(start_column, end_column), position_names):
                     try:
                         player_data = line[index]
+                        if player_data.find('(LOCKED)') >= 0: # Remove possible '(LOCKED)' substring
```
Comment (DimaKudosh) [inline pydfs_lineup_optimizer/sites/draftkings/classic/importer.py]: You can simply make `player_data = player_data.replace('(LOCKED)', '')` for removing locked text

File: pydfs_lineup_optimizer/sites/draftkings/classic/importer.py (hunk 1 above)
Comment (utahman) [reply pydfs_lineup_optimizer/sites/draftkings/classic/importer.py]: Its possible that they extend comments and move the beginning of data to line 9 or lower. Searching for the line makes it more resilient to changes.
Another reason is that my scripts do not use line 8.
I'll add the fix for possible infinite loop.

**Pull request merged**